common/src/misc/utils.ts

```diff
@@ -221,6 +221,11 @@ export class Utils {
         if (httpUrl) {
             try {
                 const url = Utils.getUrlObject(uriString);
+                const validHostname = tldjs?.isValid != null ? tldjs.isValid(url.hostname) : true;
```

Comment:
Do we need to update our noop implementation as well? (Always return true perhaps)